### ISSUE TITLE
Rework supported events page

### ISSUE THREAD

**Issue by incertum** [falcosecurity/falco-website#1031]
/area documentation

This page could benefit from some updates since the introduction of the adaptive syscalls feature.
In the blogpost we link to the repo readme for a list of syscalls, perhaps this is a more maintainable strategy? But it's not versioned and could be out of sync with the current release, therefore maybe link to the corresponding libs release branch?

https://github.com/falcosecurity/libs/blob/master/driver/report.md

**What would you like to be added**:

https://falco.org/docs/reference/rules/supported-events/

**Why is this needed**:

**Comment by incertum** [comment]
Also related to https://github.com/falcosecurity/falco-website/issues/985 to somehow make it clearer to go to that page to retrieve the `evt.arg.*` fields for Falco rules output fields? Or do we want to add the args to the report markdown table?

**Comment by leogr** [comment]
The [supported event page](https://falco.org/docs/reference/rules/supported-events/) has been recently improved by https://github.com/falcosecurity/falco-website/pull/1068 cc @LucaGuerra 

@incertum do you think anything is still missing?

Also, the supported syscalls report has been moved to https://falcosecurity.github.io/libs/report/

**Comment by leogr** [comment]
> Awesome! re the new https://falcosecurity.github.io/libs/report/ we may need to do a search for places we link to the libs report.md and replace the links or now consistently link to the reworked supported events wepage?

Good catch! Fixed by https://github.com/falcosecurity/falco-website/pull/1096/commits/19a31970dd2b5301e79830707c382464d8a07700 (not yet merged)
